go/riscv64.patch

```diff
@@ -28,3 +36,23 @@ index cece4cc..d368d3c 100644
  
    cp -a bin pkg src lib misc api test "$pkgdir/usr/lib/go"
    # We can't strip all binaries and libraries,
+diff --git cacheline.patch cacheline.patch
+new file mode 100644
+index 0000000..1b3e98e
+--- /dev/null
++++ cacheline.patch
+@@ -0,0 +1,14 @@
++diff --git a/src/internal/cpu/cpu_riscv64.go b/src/internal/cpu/cpu_riscv64.go
++index 54b8c3378b4ad..2173fe8886090 100644
++--- a/src/internal/cpu/cpu_riscv64.go
+++++ b/src/internal/cpu/cpu_riscv64.go
++@@ -4,7 +4,7 @@
++ 
++ package cpu
++ 
++-const CacheLinePadSize = 32
+++const CacheLinePadSize = 64
```

Comment:
Could you please give a brief explanation of the reason of this change? It will help us reviewing your patch, and will keep as a document for other contributors.